Sources/SwiftParserDiagnostics/DiagnosticExtensions.swift

```diff
@@ -173,4 +173,15 @@ extension FixIt.MultiNodeChange {
 
     return FixIt.MultiNodeChange(primitiveChanges: changes)
   }
+
+  /// Make `oldNode` replaced by `newNode` but maintain `oldNode`'s trivia, where `oldNode` should be a missing node
+  static func makeReplacingPresentWithTrivia(oldNode: some SyntaxProtocol, newNode: some SyntaxProtocol) -> Self {
```

Comment:
I think the comment is a little hard to understand and I couldn’t come up with a better explanation of what this function does, which I often take for a sign that it might not be the right API design. What do you think about the following?

```swift
/// Make `node` present while changing its content to `tokenKind`.
static func makePresent(_ node: some SyntaxProtocol, with tokenKind: TokenKind)
```

Reply:
I think that's nice. I made `node` a `TokenSyntax` since I think this function should only be used by presenting a `TokenSyntax` and changing its token kind.